scripts/tools.go

```diff
@@ -1,4 +1,7 @@
+//go:build ex
```

Comment:
so this makes sure that this package does not end up in the binary?

Reply:
Not exactly.  These tool-import files are a bit of a hack to get go mod to fetch executables.  Unfortunately, `go test` can recognize this, but will still fail.  So slapping an arbitrary build constraint is an idiomatic way to enable `go test ./...`.  I meant to remove this one however, since it was just to make my IDE happy.